scripts-foundry/common/ProxyInitCodeHelper.s.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import {Script, console} from "forge-std/Script.sol";
+import {LibClone} from "solady/src/utils/LibClone.sol";
+
+contract ProxyInitCodeHelper is Script {
+    function run() external {
+        address implementation = address(uint160(vm.envUint("IMPLEMENTATION")));
+       
+        vm.startBroadcast();
+        
+        bytes memory proxyInitCode = LibClone.initCodeERC1967(implementation);
+        console.log("Proxy init code: %s", vm.toString(proxyInitCode));
```

Comment:
After it's console logged, then what? Is this verification? Is it copy/pasted somewhere? Is it dumped in a var and used with the cast command in another file?